tests/phpunit/includes/FakeHttpClient.php

```diff
@@ -35,11 +36,15 @@ public function set_request_handler( callable $handler ) {
 	}
 
 	/**
+	 * Fake sending an HTTP request.
+	 *
 	 * @param RequestInterface $request
+	 * @param array            $options Request options to apply to the given
+	 *                                  request and to the transfer. See \GuzzleHttp\RequestOptions.
 	 *
-	 * @return \Google\Site_Kit_Dependencies\GuzzleHttp\Message\ResponseInterface
+	 * @return ResponseInterface
 	 */
-	public function send( RequestInterface $request ) {
```

Comment:
This class is no longer useful for its intended purpose and needs to be replaced.